src/App.jsx

```diff
@@ -52,9 +54,9 @@ function App() {
 
   const editorRef = useRef(null);
 
-  const onDataChange = (event) => {
+  const onDataChange = (data) => {
     try {
-      const raw = event.target.value;
+      const raw = data;
```

Comment:
Why not just `const json = JSON.raw(data);` ?

Reply:
That is a good point, I'll update this now 👍